presto-hive/src/main/java/io/prestosql/plugin/hive/s3/S3SecurityMappingConfig.java

```diff
@@ -27,6 +27,20 @@
     private String roleCredentialName;
     private Duration refreshPeriod;
     private String colonReplacement;
+    private String securityMappingsProvider;
```

Comment:
Use `enum` instead of `String` with values like `FILE`.

Reply:
This is relevant under the conditions that all implementations of `S3SecurityMappingsProvider` are part of `presto-hive`, right?